tests/Unit/Repositories/CategoryTest.php

```diff
@@ -0,0 +1,69 @@
+<?php
+
+namespace Tests\Unit\Repositories;
+
+use Illuminate\Database\Eloquent\Collection;
+use IndieHD\Velkart\Contracts\ProductRepositoryContract;
+use IndieHD\Velkart\Contracts\CategoryRepositoryContract;
+use IndieHD\Velkart\Tests\Unit\Repositories\RepositoryTestCase;
+
+class CategoryTest extends RepositoryTestCase
+{
+    protected $productRepository;
+
+    public function setUp(): void
+    {
+        parent::setUp();
+
+        $this->repo = resolve(CategoryRepositoryContract::class);
+
+        $this->productRepository = resolve(ProductRepositoryContract::class);
+    }
+
+    /** @test */
+    public function itCanUpdateACategory()
+    {
+        $category = $this->create();
+
+        $updated = $this->repo->update($category->id, [
+            'name' => $category->name . ' new',
+        ]);
+
+        $this->assertTrue($updated, 'Category did NOT update');
+        $this->assertDatabaseHas('categories', ['name' => $category->name . ' new']);
```

Comment:
should change all static table names to dynamic for example:

```php
$this->assertDatabaseHas($this->repo->model()->getTable(), ['name' => $category->name . ' new']);
```

This will allow it to remain dynamic